README.md

```diff
@@ -62,3 +62,350 @@ app/assets/stylesheets/application.scss
 ```
 $ yarn add react-router-dom@5.3.0
 ```
+### Feb/19/2022
+> made a branch called 'index'
```

Comment:
I don't think you need to check these updates in to the README. The PR itself essentially serves as a log of work done. You could move your comments here into the description of the PR. Then, even once the PR is merged, you can see it in the closed PR's tab.